contracts/src/v0.8/automation/upkeeps/LinkAvailableBalanceMonitor.sol

```diff
@@ -36,33 +36,66 @@ interface ILinkAvailable {
      we could save a fair amount of gas and re-write this upkeep for use with Automation v2.0+,
      which has significantly different trust assumptions
  */
-contract LinkAvailableBalanceMonitor is ConfirmedOwner, Pausable, KeeperCompatibleInterface {
+contract LinkAvailableBalanceMonitor is ConfirmedOwner, Pausable, AutomationCompatibleInterface {
   using EnumerableMap for EnumerableMap.AddressToUintMap;
 
+  /**
```

Comment:
Can we use `//` over `/* */` comments? Saves 66% of the lines in these cases and is part of the new style guide :)

Reply:
Maybe controversial, but I don't feel like these comments actually add much over just the name of the event so I'd drop them

Reply:
Dropped unnecesary comments.